server/src/main/java/io/crate/metadata/view/ViewInfo.java

```diff
@@ -84,7 +84,7 @@ public Settings parameters() {
 
     @Override
     public Set<Operation> supportedOperations() {
-        return Operation.READ_ONLY;
+        return EnumSet.of(Operation.READ, Operation.ALTER_TABLE_RENAME);
```

Comment:
Does this have any impact? is it a fix on behavior, or it was revealed only by the other changes on this PR?

Reply:
No functional change or fix. The old `resolveRelation` didn't check the `supportedOperations`, so renaming a view worked despite it missing from `supportedOperations`.

Now findRelation checks the `supportedOperations`  so it must be present to keep rename working.